hardhat/common-contracts.js

```diff
@@ -6,41 +6,58 @@ const fs = require('fs');
 const path = require('path');
 
 const INSTANCES = {
-  // Entrypoint v0.7.0
+  // ERC-4337 Entrypoints
   entrypoint: {
-    address: '0x0000000071727De22E5E9d8BAf0edAc6f37da032',
-    abi: JSON.parse(fs.readFileSync(path.resolve(__dirname, '../test/bin/EntryPoint070.abi'), 'utf-8')),
-    bytecode: fs.readFileSync(path.resolve(__dirname, '../test/bin/EntryPoint070.bytecode'), 'hex'),
+    v07: {
+      address: '0x0000000071727De22E5E9d8BAf0edAc6f37da032',
+      abi: JSON.parse(fs.readFileSync(path.resolve(__dirname, '../test/bin/EntryPoint070.abi'), 'utf-8')),
+      bytecode: fs.readFileSync(path.resolve(__dirname, '../test/bin/EntryPoint070.bytecode'), 'hex'),
+    },
+    v08: {
+      address: '0x4337084D9E255Ff0702461CF8895CE9E3b5Ff108',
+      abi: JSON.parse(fs.readFileSync(path.resolve(__dirname, '../test/bin/EntryPoint080.abi'), 'utf-8')),
+      bytecode: fs.readFileSync(path.resolve(__dirname, '../test/bin/EntryPoint080.bytecode'), 'hex'),
+    },
   },
   senderCreator: {
-    address: '0xEFC2c1444eBCC4Db75e7613d20C6a62fF67A167C',
-    abi: JSON.parse(fs.readFileSync(path.resolve(__dirname, '../test/bin/SenderCreator070.abi'), 'utf-8')),
-    bytecode: fs.readFileSync(path.resolve(__dirname, '../test/bin/SenderCreator070.bytecode'), 'hex'),
+    v07: {
+      address: '0xEFC2c1444eBCC4Db75e7613d20C6a62fF67A167C',
+      abi: JSON.parse(fs.readFileSync(path.resolve(__dirname, '../test/bin/SenderCreator070.abi'), 'utf-8')),
+      bytecode: fs.readFileSync(path.resolve(__dirname, '../test/bin/SenderCreator070.bytecode'), 'hex'),
+    },
+    v08: {
+      address: '0x449ED7C3e6Fee6a97311d4b55475DF59C44AdD33',
+      abi: [], // not verified yet
```

Comment:
Seems this is already verified:
https://etherscan.io/address/0x449ED7C3e6Fee6a97311d4b55475DF59C44AdD33#code